selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/theia/TheiaBuildPluginTest.java

```diff
@@ -133,4 +252,30 @@ private void openTerminal(String topMenuCommand, String commandName, String prop
     theiaProposalForm.clickOnProposal(proposalText);
     theiaProposalForm.waitFormDisappearance();
   }
+
+  private void waitNotificationEqualsTo(String notificationMessage, String parentWindowHandle) {
+    seleniumWebDriverHelper.waitSuccessCondition(
+        driver -> {
+          if (seleniumWebDriverHelper.isOpenedSomeWin()) {
+            seleniumWebDriver.switchTo().window(parentWindowHandle);
```

Comment:
It's better to avoid direct usage of SeleniumWebDriver instance usage in the tests.